lib/pbench/test/unit/agent/task/test_results_push.py

```diff
@@ -148,64 +145,36 @@ def test_multiple_meta_args_single_option():
             ],
         )
         assert result.exit_code == 0, result.stderr
-        assert result.stderr == "File uploaded successfully\n"
-
-    @staticmethod
-    @responses.activate
-    def test_multiple_meta_args():
-        """Test normal operation when all arguments and options are specified"""
-
-        TestResultsPush.add_http_mock_response()
-        runner = CliRunner(mix_stderr=False)
-        result = runner.invoke(
-            main,
-            args=[
-                TestResultsPush.TOKN_SWITCH,
-                TestResultsPush.TOKN_TEXT,
-                TestResultsPush.ACCESS_SWITCH,
-                TestResultsPush.ACCESS_TEXT,
-                TestResultsPush.META_SWITCH,
-                TestResultsPush.META_TEXT_TEST,
-                TestResultsPush.META_SWITCH,
-                TestResultsPush.META_TEXT_FOO,
-                tarball,
-            ],
-        )
-        assert result.exit_code == 0, result.stderr
-        assert result.stderr == "File uploaded successfully\n"
```

Comment:
Why was this test removed?

Reply:
I'm not sure: I guess because I thought it was just a "normal" test and missed the separate  `meta` switches. I'll need to add it back both here and on main.